test/static/quantity_test.cpp

```diff
@@ -937,7 +937,11 @@ static_assert(is_of_type<quantity_cast<isq::distance>(1 * m), quantity<isq::dist
 static_assert(is_of_type<quantity_cast<isq::distance>(isq::length(1 * m)), quantity<isq::distance[m], int>>);
 static_assert(is_of_type<quantity_cast<kind_of<isq::length>>(isq::length(1 * m)), quantity<si::metre, int>>);
 static_assert(is_of_type<quantity_cast<kind_of<isq::length>>(isq::distance(1 * m)), quantity<si::metre, int>>);
-
+// lvalue references in quantity_cast
+inline constexpr quantity<isq::distance[m], int> to_distance(quantity<m, int> arg)
```

Comment:
`inline` is not needed here.

Reply:
Also, I am not sure if we need a function here as it is not a test (although it has to be compiled correctly).
Maybe it would be better to define a quantity variable and pass it to a `static_assert()`. With this, you could check it the output type is exactly what you require as well.